### PR TITLE
Update CHANGELOG for 3.4.0

### PR DESCRIPTION
**Pull request by tunetheweb**
Let's cut a 3.4.0 with the latest fixes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+### v3.4.0 (2023-07-11)
+
+- Make bindReporter generic over metric type ([#359](https://github.com/GoogleChrome/web-vitals/pull/359))
+- Update INP status in README ([#362](https://github.com/GoogleChrome/web-vitals/pull/362))
+- Metric type fixes ([#356](https://github.com/GoogleChrome/web-vitals/pull/356))
```
Comment (philipwalton) [inline CHANGELOG.md:7]: Nit: can this be reworded to be consistent with the other entries (e.g. where the first word is an active verb)?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+- Fix selector for SVGs ([#354](https://github.com/GoogleChrome/web-vitals/pull/354))
```
Comment (philipwalton) [inline CHANGELOG.md:8]: I think it'd be helpful if this clarified that it applies to selector identifiers *just* in the attribution build.

Ideally someone could glance at these CHANGELOG entries and quicky see (without clicking through to the PR) whether a change applies to them.

**Review by philipwalton — APPROVED** [review]
LGTM, with a few suggestions for how to make the entry descriptions a bit more clear.

FWIW, in the past, I would sometimes change the entry description from the original PR title (or add additional text) to make it more clear so someone could quickly see at a glance whether or not a change might affect them.

File: CHANGELOG.md (hunk 1 above)
Comment (tunetheweb) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (tunetheweb) [reply CHANGELOG.md]: Done.

**Review by philipwalton — APPROVED** [review]

**Pull request merged**